CommunityProject/Assets/_Game/Scripts/Runtime/Utilities/MathfUtilities.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using UnityEngine;
+
+namespace BoundfoxStudios.CommunityProject.Utilities
+{
+	public static class MathfUtilities
+	{
+		public static float Round(float value, int decimals)
```

Comment:
```suggestion
		public static float RoundToDecimalPlaces(float value, int decimals)
```
Vllt so etwas besser vom Naming her von den anderen Round Methoden zu unterscheiden.
Ansonsten XML Kommentar :)
Wieso an der Stelle eigtl nicht auch eine ExtensionMethod? Um deutlicher zu machen das Mathf hintendran genutzt wird?

Reply:
> Vllt so etwas besser vom Naming her von den anderen Round Methoden zu unterscheiden.

Angelehnt an [System.Math.Round](https://learn.microsoft.com/en-us/dotnet/api/system.math.round?view=net-7.0) :)
Kommentar ist aber eine gute Idee!

> Wieso an der Stelle eigtl nicht auch eine ExtensionMethod? Um deutlicher zu machen das Mathf hintendran genutzt wird?

Hm, eigentlich hast Recht. Ich schreib's als Extension Method um.
Ich dachte eigentlich, dass es so rum besser ist, da man ja auch `Math.` und `Mathf.` benutzt. Auf der anderen Seite müsste ich dann die Extension-Methoden vom Vector wieder runternehmen auch ein VectorUtils basteln.